### PR TITLE
Replacing vng by anatal

### PR DESCRIPTION
**Pull request by andrenatal**

### CODE REVIEW

**Comment by andrenatal** [comment]
@haroldwoo r?

**Comment by haroldwoo** [comment]
also do the taar_weekly dag

**Review by haroldwoo — APPROVED** [review]
lgtm feel free to merge once you do the same for taar_weekly.py

**Comment by andrenatal** [comment]
@haroldwoo I don't have merge permissions :(

**Pull request merged**
